projects/storefrontlib/src/cms-structure/services/cms-routes.service.ts

```diff
@@ -40,28 +40,33 @@ export class CmsRoutesService {
   handleCmsRoutesInGuard(
     pageContext: PageContext,
     componentTypes: string[],
-    currentUrl: string
+    currentUrl: string,
+    currentPageLabel: string
```

Comment:
Note: it's a private service (not exported from library), to it's not a breaking change.